replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+---
+title: "replication"
+author: "Nick F & Aaron K"
+date: "March 15, 2015"
+output: html_document
+---
+
+```{r echo = FALSE}
+suppressPackageStartupMessages(require(dplyr))
+suppressPackageStartupMessages(require(tidyr))
+suppressPackageStartupMessages(require(aod)) # wald.test
+suppressPackageStartupMessages(require(broom))
+suppressPackageStartupMessages(require(lsr)) #etaSquared
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(foreign))
+
+# detect the user's directory:
+directory  <- "C:/Users/Nick/Documents/Github/scarcityvalueRep"
+if (Sys.getenv('USER')=="air") directory <- "~/Dropbox/SOC504/replicate"
+setwd(directory)
+
+# Function : calculates standard error of the mean
+Sem <- function(x) {
+  sqrt(var(x)/length(x))
+  }
+
+# Function : to generate mean and confidence interval & name
+Mean_CI <- function(title, x){
+  print(title, quote=F)
+  # print Mean
+  M <- t.test(x) %>%
+    .$estimate %>%
+    round(digits = 2) %>%
+    .[[1]]
+  print(c("M:", M), quote = F)
+  # print Confidence Interval 95%
+  CI <- t.test(x) %>%
+    .$conf.int %>%
+    round(digits = 3) %>%
+    .[1:2]
+  print(c("CI : ", CI), quote = F)
+}
+```
+<br> <br> <br>
+*note: beta coefficients for linear regressions calculate consistantly larger than published, even though the t value and p values are as published.  This occurs several times throughout. t values that match the publication are available in the models.*
+<br> <br>
+*Update 4/25/15 - The unmatched beta coefficients are due to the publication reporting standardized coefficients, and R function lm reporting unstandardized beta coefficients*
+<br> <br>
+*Update 4/26/15 - unmatched beta coefficients have been fixed using the scale() function.*
+
```

Comment:
Your comments could be a bit more parsimonious  - rather than updating with a timestamp (since git will track changes itself) you can make your commenting more concise and reflect your most current conclusions/observations.